vars/withSecrets.txt

```diff
@@ -4,3 +4,7 @@ It must be called while the current directory is the workspace-root.
 
 NOTE(benkraft): withSecrets is leaky; other branches of the same job will see
 the secrets file too, even if they are not in a withSecrets block.
+
+We also provide the ability to just decrypt one secret: the slack
```

Comment:
Should this also be a TODO?

Reply:
I see it as just describing the function that only decrypts one secret, which actually is part of this PR.  The fact the function does extra work isn't really relevant to this help text...